soh/src/overlays/actors/ovl_En_Reeba/z_en_reeba.c

```diff
@@ -155,6 +155,7 @@ void EnReeba_Destroy(Actor* thisx, PlayState* play) {
         if (spawner->actor.update != NULL) {
             if (spawner->curNumSpawn > 0) {
                 spawner->curNumSpawn--;
+                spawner->timer = 1 * (CVarGetInteger("gLeeverSpawnRate", 0) * 10);
```

Comment:
```suggestion
                spawner->timer = 1 * (CVarGetInteger("gLeeverSpawnRate", 1) * 10);
```

if it defaults to zero then when it isn't set/in the json the timer will always be zero